iBudget/spending_history/views.py

```diff
@@ -23,16 +23,20 @@ def register_spending(request):
     data = json.loads(request.body)
     if input_spending_registration_validate(data):
         return HttpResponse(status=400)
+
     owner = request.user
-    fund = FundCategories.get_by_id(int(data["type_of_pay"]))
     spending = SpendingCategories.get_by_id(int(data["category"]))
-
     if spending.owner == owner:
-        SpendingHistory.create(fund,
-                               spending,
-                               owner,
-                               Decimal(data["sum"]),
-                               data["date"],
-                               data["comment"])
-        return HttpResponse(status=201)
+        spending_history = SpendingHistory()
+        spending_history.fund = FundCategories.get_by_id(int(data["type_of_pay"]))
```

Comment:
You can just use `spending_history.fund_id`

Reply:
If you have id

Reply:
I refactoring this method